src/lib/zkapps_examples/zkapps_examples.ml

```diff
@@ -261,7 +261,11 @@ module Account_update_under_construction = struct
         ; use_full_commitment = Boolean.false_
         ; caller = t.caller
         ; authorization_kind =
-            { is_signed = Boolean.false_; is_proved = Boolean.true_ }
+            (* TODO: is there a valid vk hash available? *)
+            { is_signed = Boolean.false_
+            ; is_proved = Boolean.true_
+            ; verification_key_hash = Field.zero
```

Comment:
This seems like it should cause the test to fail.. I think we're missing a check somewhere :thinking: